pkgs/applications/window-managers/sway/default.nix

```diff
@@ -33,11 +33,12 @@ stdenv.mkDerivation rec {
   buildInputs = [
     wayland libxkbcommon pcre json_c dbus libevdev
     pango cairo libinput libcap pam gdk-pixbuf librsvg
-    wlroots wayland-protocols
+    wlroots wayland-protocols libdrm
   ];
 
   mesonFlags = [
     "-Ddefault-wallpaper=false"
+    "-Dauto_features=auto"
```

Comment:
We should avoid that. Features that we don't want should be disabled explicitly.

Reply:
```suggestion
    "-Dsd-bus-provider=libsystemd"
```